studio/app/optinist/wrappers/caiman/cnmf.py

```diff
@@ -116,9 +118,12 @@ def caiman_cnmf(
         backend="local", n_processes=None, single_thread=True
     )
 
-    cnm = cnmf.CNMF(n_processes=n_processes, dview=dview, Ain=None, params=ops)
+    cnm = cnmf.CNMF(n_processes=n_processes, dview=dview, Ain=Ain, params=ops)
     cnm = cnm.fit(mmap_images)
 
+    if params.get("do_refit"):
+        cnm = cnm.refit(mmap_images, dview=dview)
```

Comment:
@quanpython 
Although the causal relationship is unclear, the following WARNING is generated when refit is used. (Confirmed on mac, window)

```
D:\workspace\barebone-studio\.snakemake\conda\9010a07d49a2beaf662d3cc55fe23c32_\lib\site-packages\sklearn\linear_model\_base.py:121:
  FutureWarning: 'normalize' was deprecated in version 1.2 and will be removed in 1.4.
  Please leave the normalize parameter to its default value to silence this warning.
  The default behavior of this estimator is to not do any normalization.
  If normalization is needed please use sklearn.preprocessing.StandardScaler instead.
```

Please investigate.
*But the refit process itself seems to be successful, so if it seems to take time to investigate and resolve, please consult us.

Reply:
@itutu-tienday
The above warning shows that 'normalize' of the scikit-learn library was deprecated at version 1.2.
I check my environment and scikit-learn is on 1.3 with Caiman 1.9.9.
Since it is a dependency of the Caiman library, it will be difficult to intervene.

Reply:
@quanpython 
Understood.
If the warning is not a problem for processing, it will be a separate remaining issue, and we will close it once here.